packages/dashmate/docker-compose.platform.build.yml

```diff
@@ -4,7 +4,8 @@ services:
   drive_abci:
     build:
       context: ${PLATFORM_SOURCE_PATH:?err}
-      dockerfile: ${PLATFORM_SOURCE_PATH:?err}/packages/rs-drive-abci/Dockerfile
+      dockerfile: ${PLATFORM_SOURCE_PATH:?err}/Dockerfile
+      target: drive-abci
       args:
         SCCACHE_MEMCACHED: ${SCCACHE_MEMCACHED}
```

Comment:
Cool, this is much better! Let's rip out https://github.com/dashpay/platform/blob/15b3ac912d34736ef4342db46280dddfe66ab29c/packages/dashmate/src/listr/tasks/buildServicesTaskFactory.js#L23